cg/store/models.py

```diff
@@ -442,20 +444,52 @@ class Sample(Model, PriorityMixin):
     ordered_at = Column(types.DateTime, nullable=False)
     organism_id = Column(ForeignKey("organism.id"))
     organism = orm.relationship("Organism", foreign_keys=[organism_id])
+    _phenotype_terms = Column(types.Text)
     prepared_at = Column(types.DateTime)
     priority = Column(types.Integer, default=1, nullable=False)
     reads = Column(types.BigInteger, default=0)
     received_at = Column(types.DateTime)
     reference_genome = Column(types.String(255))
     sequence_start = Column(types.DateTime)
     sequenced_at = Column(types.DateTime)
+    _synopsis = Column(types.Text)
     sex = Column(types.Enum(*SEX_OPTIONS), nullable=False)
     ticket_number = Column(types.Integer)
     time_point = Column(types.Integer)
 
     def __str__(self) -> str:
         return f"{self.internal_id} ({self.name})"
 
+    @property
```

Comment:
I suggest you check out Sourcery plugin for pycharm! 

To avoid many confusing unused variables, maybe you could return all the objects in `@property ` functions directly?

Reply:
I installed the plugin and will give it a try, thanks!

returns directly now 👍